08_files_lab/01.py

```diff
@@ -1,6 +1,16 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+_author_ = 'mariag'
+
+from sys import argv
+
+if len(argv)!= 3:
+    print 'usage: <01.py> <file_name1> <file_name2>'
+    exit(1)
+
+file_name1 = argv[1]
+file_name2 = argv[2]
+
+with open(file_name1, 'r') as f_from:
+    with open(file_name2, 'a') as f_to:
+        f_to.write('\n')
+        for line in f_from:
+            f_to.write(line)
```

Comment:
Nice, though I'd probably use more descriptive variable names instead of file_name1 and file_name2 just as you used for the open file handles f_from and f_to. 👍